tools/tertiary-analysis/sc3/sc3-sc3-calc-biology.xml

```diff
@@ -1,17 +1,18 @@
-<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="0.1.0">
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="@TOOL_VERSION@+galaxy0">
     <description>SC3 calculates biologically relevant clusters</description>
     <macros>
-        <import>macros.xml</import>
+        <import>sc3-macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
         sc3-sc3-calc-biology.R -i "$input1" -o "$output1"
     ]]></command>
     <inputs>
-        <param type="data" name="input1" format="rds" />
+        <param type="data" name="input1" format="rds" label="File name in which a SC3 'SingleCellExperiment' object has been stored after kmeans clustering"/>
     </inputs>
     <outputs>
-        <data name="output1" format="rds" />
+        <data name="output1" format="rds" label="File name for R object of type 'SingleCellExperiment' from SC3 in which to store the consensus matrix"/>
```

Comment:
Please use rdata instead of RDS. My understanding is that outputs don't have labels as far as I know.